app/javascript/sdk/IFrameHelper.js

```diff
@@ -56,7 +56,8 @@ export const IFrameHelper = {
       widgetUrl = `${widgetUrl}&cw_conversation=${cwCookie}`;
     }
     iframe.src = widgetUrl;
-
+    iframe.allow =
```

Comment:
Added this to make sure that the website LiveChat widget asks for permissions when the SDK is loaded.